### DISCUSSION TITLE
【灌水】不知道大家抽椒丘了吗

### DISCUSSION THREAD

**Discussion by wenkui921** [share121/inter-knot#3441] [其他]
嘿嘿，这是我昨天（过12点了应该算昨天）中午90发小保底出的椒椒
没歪真是太好了呢(˶´˘`˶)
![Screenshot_20240821_131119_com miHoYo hkrpg](https://github.com/user-attachments/assets/f5896074-869c-483f-b00f-8887851268b9)

**Comment by HyperMuteki0347** [comment]
首先恭喜，但我忍着抽，因为这是大捷捷对我的考验<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/8b9838175cb6312df34c32648a8e6649_1050729359995605746.png" alt="景元 吸猫" width="75" >

**Comment by Fooler00** [comment]
木有抽，我自己是二命黄泉，感觉花火更适合，不知道感觉的对不对
![img_-2079396521_1719347327516](https://github.com/user-attachments/assets/3a8934e7-f914-4e69-97f5-b05a0a64962a)

**Comment by Guessme404** [comment]
100抽1命+专武<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/01e64050e1304a68e0234639bbdb7188_855263133676056799.png" alt="彦卿 哼" width="75" >

**Comment by lllliufeng** [comment]
五星，丰饶，稳辣！
什么？椒丘是虚无角色？
丸辣！（

**Comment by HousiqiGithub** [comment]
有鹅（奥迹yyds）了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
